README.md

```diff
@@ -16,15 +16,15 @@ Mendix 7.2
 
 [https://badgebutton.mxapps.io/](https://badgebutton.mxapps.io/)
 
-![1](https://raw.githubusercontent.com/mendixlabs/badge-button/v1.1.0/assets/demo.png)
+![1](https://raw.githubusercontent.com/mendixlabs/badge-button/v1.1.1/assets/demo.png)
```

Comment:
The `URL` has to be relative so that it can show in the mendix app store eg
![Demo](/assets/demo.png)